site/_includes/partials/devsite-header.njk

```diff
@@ -0,0 +1,41 @@
+{% from 'macros/icon.njk' import icon with context %}
+
+<top-nav role="banner" class="display-block hairline-bottom" data-side-nav-inert>
+  <a href="#main-content" class="color-primary visually-hidden skip-link">{{ 'i18n.common.skip_to_content' | i18n(locale) }}</a>
+
+  <nav class="display-flex align-center" aria-label="{{ 'i18n.common.chrome_developers' | i18n(locale) }}" role="search">
+    {# Hamburger button #}
+    <button class="dcc-hamburger button display-flex align-center justify-content-center width-700 height-700" aria-label="{{ 'i18n.common.menu' | i18n(locale) }}">
+      {{ icon('hamburger') }}
+    </button>
+
+    {# Logo #}
+    {# This is referenced by the sidebar in 'navigation-rail.njk', don't change these IDs or SVGs
+       without also updating there #}
+    <div class="top-nav__logo display-flex justify-content-start">
+      <a class="display-inline-flex" href="/" aria-label="developer.chrome.com">
+        {{ icon('logo', {id: 'chromeLogo', hidden: true}) }}
+        <div class="logo__text">
+          {{ icon('logo-lockup', {id: 'chromeDevelopersText', hidden: true}) }}
+        </div>
+      </a>
+    </div>
+
+    <nav class='dcc-top-menu'>
+      <a href='/' class='no-visited'>Get Inspired</a>
```

Comment:
Please no untraslatable strings in template

Reply:
Hi Ewa, sorry we don't quite understand what this means. Where are the no untranslatable strings in the lines that you have requested change?